dts/bindings/ethernet/nxp,tja1103-phy.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: TJA1103 PHY
+
+compatible: "nxp,tja1103-phy"
```

Comment:
All TJAxxxx devices are PHY, either CAN or Ethernet. I'd suggest to name it just `nxp,tja1103`.

Reply:
Makes sense, done.